cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -36,6 +36,7 @@ const (
 	sysfsDrmDirectory = "/sys/class/drm"
 	devfsDriDirectory = "/dev/dri"
 	gpuDeviceRE       = `^card[0-9]*$`
+	controlDeviceRE   = `^controlD[0-9]*$`
```

Comment:
This regexp has typo, which actually this also uncovered bug in the line above. it should be `^xxxxxxx[0-9]+$`